config/flink.batch.conf.template

```diff
@@ -26,9 +26,10 @@ env {
 
 source {
   # This is a example input plugin **only for test and demonstrate the feature input plugin**
-  FileSource{
-    file.path = "hdfs://localhost:9000/output/text"
-    source_format = "text"
+  FileSource {
+    path = "/opt/data.txt"
+    format.type = "text"
+    schema = "text"
```

Comment:
It seems unnecessary to write `schema = "text"` if `format.type` is `text`

```java
            case "text":
                TextRowInputFormat textInputFormat = new TextRowInputFormat(filePath);
                inputFormat = textInputFormat;
                break;
```

Reply:
Make sense, thanks